tools/MultiProjectValidator/project.json

```diff
@@ -5,7 +5,7 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "Microsoft.NETCore.App": "1.0.0-rc3-*",
+    "Microsoft.NETCore.App": "1.0.0-rc2-*",
```

Comment:
We don't have Microsoft.NETCore.App RC3 packages in the CLI-DEPS feed. Also, the tool has absolutely no need to be on the latest package for this. Either we have the tool on the rc2 version, or we go back do depending on the dotnet core feed. I prefer this.